gitlab/README.md

```diff
@@ -101,7 +101,9 @@ Collecting logs is disabled by default in the Datadog Agent. To enable it, see [
 
 ### Metrics
 
-See [metadata.csv][12] for a list of metrics provided by this integration.
+See [metadata.csv][12] for a list of metrics provided by this integration. 
+
+**Note**: Metrics marked as `[Legacy]`, `[OpenMetricsV1]`, or `[OpenMetricsV2]` are only available using this implementation of the GitLab integration. Metrics not marked are collected by all three implementations.
```

Comment:
Just a tiny nit, would you say this is a little bit clearer?

```suggestion
**Note**: Metrics marked as `[Legacy]`, `[OpenMetricsV1]`, or `[OpenMetricsV2]` are only available using the corresponding implementation of the GitLab integration. Metrics not marked are collected by all three implementations.
```

Reply:
Good call! I like it